.dockerignore

```diff
@@ -2,23 +2,27 @@
 *
 
```

Comment:
Could you comment on how these changes came about?

Reply:
I basically rearranged them in alphabetical order, and then added some folders and files that were needed by cloudtest.

Reply:
I was using a dockerfile for local testing on my mac before I started using the cisco-hosted-jenkins environment.